Sources/XMTPiOS/SigningKey.swift

```diff
@@ -24,10 +24,10 @@ public protocol SigningKey {
 	var isSmartContractWallet: Bool { get }
 	
 	/// The name of the chainId for example "1"
-	var chainId: Int64? { get }
+	var chainId: UInt64? { get }
```

Comment:
I actually think we should keep these Int64 and change the RN to Int64. I do all the U things to it in the code already so no need for these to be UInt64 here.

Reply:
Yea I changed this here for two reasons

- Passing the ReactNativeSigner as a SigningKey doesn't work as the chainId getter have different types. So needed to align those.
- When aligning I choose to stick with the eip155 definition of chainId as unsigned

I don't think it matters that much either way so whatever works 👍